kvmagent/kvmagent/plugins/nvram/nvram_sblk.py

```
@@ -0,0 +1,62 @@
+    return "sanlock" in lock_type
+
+def deactivate_sharedblock_nvram_volume_if_needed(dev_path, vm_uuid):
+    # type: (str, str) -> None
+    logger.debug("deactivating sharedblock nvram volume %s" % dev_path)
+
+    used_process = linux.linux_lsof(dev_path)  # type: str
+    if not used_process:
+        try:
+            lvm.deactive_lv(dev_path, False)
+            logger.debug("deactivated sharedblock nvram volume %s for happened on vm %s success" % (dev_path, vm_uuid))
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)